renovate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "schedule": "before 5am every weekday",
+  "extends": [
+    "config:best-practices",
+    ":automergeDisabled",
+    ":gitSignOff"
+  ],
+  "baseBranches": [
+    "master"
+  ],
+  "enabledManagers": [
+    "dockerfile",
+    "github-actions",
+    "gomod",
+    "helm-values"
+  ],
+  "ignorePaths": [
+    "(^|/)([Dd]ocker|[Cc]ontainer)file[^/]*\\.dockerignore$"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "packageRules": [
+    {
+      "description": "go-control-plane v0.12.0 introduced a potential deadlock issue. This issue is being tracked in https://github.com/envoyproxy/go-control-plane/issues/875. Remove this once the issue is resolved",
+      "enabled": false,
+      "matchManagers": ["gomod"],
+      "matchPackageNames": ["github.com/envoyproxy/go-control-plane"]
+    },
+    {
+      "description": "No need to run tests on github actions updates",
+      "matchManagers": ["github-actions"],
+      "addLabels": ["ci/skip-test"]
```

Comment:
Why?

Reply:
I haven't spend time thinking about it, but I tried to use the same configuration as for dependabot: https://github.com/kumahq/kuma/blob/84707bb2b53296092dc5423c1a287485415cbd67/.github/dependabot.yml#L42